jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -3,7 +3,7 @@ package server
 package jetty
 
 import com.codahale.metrics.{InstrumentedExecutorService, MetricRegistry}
-import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedConnectionFactory}
+import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedHandler, InstrumentedConnectionFactory}
```

Comment:
Hmm.  [It's right here](https://github.com/http4s/http4s/blob/39d674a5d4962733ece4d3a42caacadda353be75/jetty/src/main/java/org/http4s/server/jetty/InstrumentedHandler.java).  

Can you tell if javac is running at all on that one Java file?  I don't understand why this doesn't Just Work for you, as it does for Travis and me.